item.go

```diff
@@ -0,0 +1,55 @@
+package postman
+
+type itemList []*Item
```

Comment:
after some thinking, now I believe this type should be exported (but not the method `findItem`).